instagram/src/components/Post.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import CommentSection from './CommentSection';
+
+class Post extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { }
+    }
+
```

Comment:
Add your like-handler here:

```suggestion
  handleLikes= () => {
    let likes = this.state.likes + 1;
    this.setState({ likes });
  };
```